src/common/interfaces.ts

```diff
@@ -1,9 +1,14 @@
 /* eslint-disable @typescript-eslint/naming-convention */
-
 import { GeoJSON } from 'geojson';
-import { JsonObject } from 'swagger-ui-express';
 import { CacheType, SeedMode } from './enums';
 
+type JSONValue = string | number | boolean | JSONObject | JSONArray;
+interface JSONArray extends Array<JSONValue> {}
```

Comment:
Lets talk about the need of this types in a more common library such as "https://github.com/MapColonies/types" or utils

Reply:
open an issue